frontend/webadmin/modules/uicommonweb/src/main/java/org/ovirt/engine/ui/uicommonweb/models/vms/VmListModel.java

```diff
@@ -787,6 +788,28 @@ private void edit() {
 
     }
 
+    private void confirmChangedBiosType() {
+        if (getSelectedItem() == null || ((UnitVmModel) getWindow()).getIsNew()) {
+            preSave();
+        } else {
+            VM vm = getSelectedItem();
+            UnitVmModel model = (UnitVmModel) getWindow();
```

Comment:
You can (arguably more readable) move these 2 lines to the beginning of the method and then use the variables in current line 792.